contents/tutorials/aa-testing.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How to do A/A testing
+date: 2023-07-06
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-4.png
+tags: ['experimentation', 'feature flags']
+---
+
+An A/A test is the same as an [A/B test](/docs/experiments) except both groups getting the same code or component. Because both groups get functionally identical variants, you want to see no statistical difference between the variants by the end of the experiment. If there is a significant difference, that is cause to investigate.
+
+Teams run A/A tests to ensure their A/B test service, functionality, and implementation process results in accurate and relevant results. A/A tests are a way to discover broken parts of this process, and this tutorial shows you how to run one in PostHog.
+
+> Unlike other A/B testing tools, A/A tests aren’t needed to calculate baseline conversion and sample metrics in PostHog. You can calculate those using your product data, and they are automatically when you create an experiment.
+
+## Creating an experiment
+
+The first step in running an A/A test is creating an experiment. You do this going to the [experiments tab](https://app.posthog.com/experiments) and clicking the "New experiment" button. Set a name, key, and experiment goal. Make sure to clarify in the name or description that it is an A/A test.
+
+![Create an experiment](../images/tutorials/aa-testing/experiment.png)
+
+If you use an absolute goal, like total pageviews, make sure to use an equal variant split (which is default). If you use a relative goal like conversion, you can test non-equal variant splits like 25-75. To do this, edit the feature flag with the same key as your experiment, change the variant rollout, and press save.
```

Comment:
Should we maybe just recommend the split be 50/50? It would make it simpler

```suggestion
If you use an absolute goal, like total pageviews, make sure to use an equal variant split (which is the default). If you use a relative goal like conversion, you can test non-equal variant splits like 25-75. To do this, edit the feature flag with the same key as your experiment, change the variant rollout, and press save.
```